pkg/controller/defaults/placements.go

```diff
@@ -104,29 +104,21 @@ var (
 		},
 
 		"rgw": rook.Placement{
-			NodeAffinity: &corev1.NodeAffinity{
-				RequiredDuringSchedulingIgnoredDuringExecution: &corev1.NodeSelector{
-					NodeSelectorTerms: []corev1.NodeSelectorTerm{
-						corev1.NodeSelectorTerm{
-							MatchExpressions: []corev1.NodeSelectorRequirement{
-								corev1.NodeSelectorRequirement{
-									Key:      NodeAffinityKey,
-									Operator: corev1.NodeSelectorOpExists,
+			PodAntiAffinity: &corev1.PodAntiAffinity{
+				RequiredDuringSchedulingIgnoredDuringExecution: []corev1.PodAffinityTerm{
+					corev1.PodAffinityTerm{
+						LabelSelector: &metav1.LabelSelector{
+							MatchExpressions: []metav1.LabelSelectorRequirement{
+								metav1.LabelSelectorRequirement{
+									Key:      appLabelSelectorKey,
+									Operator: metav1.LabelSelectorOpIn,
+									Values:   []string{"rook-ceph-rgw"},
 								},
 							},
 						},
+						TopologyKey: corev1.LabelHostname,
 					},
 				},
-			},
-			Tolerations: []corev1.Toleration{
```

Comment:
Why are we removing this?

Reply:
See the above comment..

Reply:
https://github.com/openshift/ocs-operator/pull/505/#discussion_r425237838

Reply:
Brought it back, with the latest changes...